packager/packager.cc

```diff
@@ -2159,6 +1963,42 @@ enum class PackagerMode {
     AllFiles,
 };
 
+class PackageDBPackageGraph {
+    core::packages::PackageDB &packageDB;
+
+public:
+    PackageDBPackageGraph(core::packages::PackageDB &packageDB) : packageDB(packageDB) {}
+
+    const vector<core::packages::Import> &getImports(core::packages::MangledName packageName) const {
```

Comment:
Is it worth using a span here to avoid accidentally copying the vector?

```suggestion
    absl::Span<const core::packages::Import> &getImports(core::packages::MangledName packageName) const {
```

Reply:
Changed to return a `Span` by value instead of `Vector` by reference.